packages/block-editor/src/components/dimensions-tool/aspect-ratio-tool.js

```diff
@@ -96,14 +35,49 @@ export default function AspectRatioTool( {
 	panelId,
 	value,
 	onChange = () => {},
-	options = DEFAULT_ASPECT_RATIO_OPTIONS,
-	defaultValue = DEFAULT_ASPECT_RATIO_OPTIONS[ 0 ].value,
+	options,
+	defaultValue = 'auto',
 	hasValue,
 	isShownByDefault = true,
 } ) {
 	// Match the CSS default so if the value is used directly in CSS it will look correct in the control.
 	const displayValue = value ?? 'auto';
 
+	const [ defaultRatios, themeRatios, showDefaultRatios ] = useSettings(
+		'dimensions.aspectRatios.default',
+		'dimensions.aspectRatios.theme',
+		'dimensions.defaultAspectRatios'
+	);
+
+	const themeOptions = themeRatios?.map( ( { name, ratio } ) => ( {
+		label: name,
+		value: ratio,
+	} ) );
+
+	const defaultOptions = defaultRatios?.map( ( { name, ratio } ) => ( {
+		label: name,
+		value: ratio,
+	} ) );
+
+	const aspectRatioOptions = [
+		{
+			label: _x(
+				'Original',
+				'Aspect ratio option for dimensions control'
+			),
+			value: 'auto',
+		},
+		...( showDefaultRatios ? defaultOptions : [] ),
+		...( themeOptions ? themeOptions : [] ),
+		...( options ? options : [] ),
```

Comment:
Unlike `trunk`, this will merge provided options instead of overriding the defaults.

Reply:
Yes and no. The defaults are now no longer hardcoded here in this JS file but instead are now coming from the core `theme.json` file. 

So the `showDefaultRatios` check determines if they display or not. The `options` currently were not used in the Gutenberg codebase at all. I now needed to use them in the unit tests since the global styles are not populated there. But I'm not sure this is breaking the expected behavior 🤔

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/47271/commits/7f6223fe1c8f24eeed53b9e83a5697006201587e